addons/event/controllers/main.py

```diff
@@ -10,6 +10,9 @@ class EventController(Controller):
 
     @route(['''/event/<model("event.event"):event>/ics'''], type='http', auth="public")
     def event_ics_file(self, event, **kwargs):
+        if request.env.user._is_public():
+            frontend_lang = request.httprequest.cookies.get('frontend_lang')
+            event = event.with_context({'lang': frontend_lang})
```

Comment:
Even if it does not crash, should we force the lang in the context to False/None instead of what is already there?